cpu/atmega1281/include/cpu_conf.h

```diff
@@ -16,8 +16,8 @@
  * @author          Hinnerk van Bruinehsen <h.v.bruinehsen@fu-berlin.de>
  */
 
-#ifndef __CPU_CONF_H
-#define __CPU_CONF_H
+#ifndef ATMEGA_CPU_CONF_H
```

Comment:
> I made this kind of revision according to the [comments](https://github.com/RIOT-OS/RIOT/pull/5571) by gebart.

I think that was meant only for the stellaris headers. RIOT's convention
so far has been FILE_NAME.H. That might be up for debate, though...